frontend/express/public/javascripts/countly/vue/components/input.js

```diff
@@ -725,6 +726,44 @@
                     classes.push("disabled");
                 }
                 return classes;
+            },
+            wrapperClasses: function() {
+                var direction = "vertical";
+                if (this.$attrs && this.$attrs.radioDirection) {
```

Comment:
I believe that using props for `radioDirection` would be a better practice here.

So you could define a `radioDirection` prop and pass a default value to it

Reply:
That would also help you to eliminate local `direction` value